roles/mariadb/tasks/main.yml

```diff
@@ -46,6 +46,17 @@
     - restart_mysql
   become: true
 
+- name: Deploy MariaDB/MySQL innodb settings file
+  copy:
+    src: 'files/innodb.cnf'
+    dest: '/etc/my.cnf.d/innodb.cnf'
```

Comment:
This new config file is located in ```/etc/my.cnf.d/``` whereas the one listed in the task above goes into ```/etc/mysql/conf.d/```; Is one of these new and the other one a deprecated location for the same thing or are these really two different locations for MySQL related config files and that must be used both?

Reply:
https://books.google.nl/books?id=wWzvCQAAQBAJ&pg=PA22&lpg=PA22&dq=%22/etc/my.cnf.d/%22+versus+%22/etc/mysql/conf.d/%22&source=bl&ots=JplxLel57M&sig=ACfU3U3ILD9J9-Zqx7Tih1jYI73WWEoJQA&hl=nl&sa=X&ved=2ahUKEwia-f3M38ToAhWDDOwKHeMfC1IQ6AEwAXoECAoQAQ#v=onepage&q=%22%2Fetc%2Fmy.cnf.d%2F%22%20versus%20%22%2Fetc%2Fmysql%2Fconf.d%2F%22&f=false

Reply:
As i understand it (from your book) differs per distribution and either one is used, never both . centos 7 uses the my.cnf.d. (I tested this)

Reply:
Ok, then ```my.cnf.d``` it is, but the task above the one you added still uses ```/etc/mysql/conf.d/my.cnf``` If that location is not used it suggests, MySQL is running with defaults for all config options that are located in ```my.cnf```...